packages/api/cms-api/src/redirects/redirects.module.ts

```diff
@@ -44,11 +44,13 @@ export class RedirectsModule {
             useValue: linkBlock,
         };
 
+        const mikroOrmModule = MikroOrmModule.forFeature([Redirect]);
+
         return {
             module: RedirectsModule,
-            imports: [MikroOrmModule.forFeature([Redirect])],
+            imports: [mikroOrmModule],
             providers: [RedirectsResolver, RedirectsDependenciesResolver, RedirectsService, linkBlockProvider, ImportRedirectsConsole],
-            exports: [RedirectsService],
+            exports: [RedirectsService, REDIRECTS_LINK_BLOCK, mikroOrmModule],
```

Comment:
Why do we need the export of the mikroOrmModule? We don't have this for the other modules

Reply:
I don't know. The import didn't work without it. Maybe because it's loaded using 
`@InjectRepository("Redirect") private readonly repository: EntityRepository<RedirectInterface>,` (vs `@InjectRepository(Redirect) private readonly repository: EntityRepository<RedirectInterface>,`)